sdk/cognitivelanguage/Azure.AI.Language.Conversations/samples/Sample5_OrchestrationPrediction.md

```diff
@@ -0,0 +1,116 @@
+
+```C# Snippet:ConversationAnalysis_AnalyzeConversationOrchestrationPredictionQnA
+string respondingProjectName = orchestratorPrediction.TopIntent;
+TargetIntentResult targetIntentResult = orchestratorPrediction.Intents[respondingProjectName];
+
+if (targetIntentResult.TargetKind == TargetKind.QuestionAnswering)
+{
+    QuestionAnsweringTargetIntentResult qnaTargetIntentResult = targetIntentResult as QuestionAnsweringTargetIntentResult;
+
+    KnowledgeBaseAnswers qnaAnswers = qnaTargetIntentResult.Result;
+
+    Console.WriteLine("Answers: \n");
+    foreach (KnowledgeBaseAnswer answer in qnaAnswers.Answers)
+    {
+        Console.WriteLine($"Answer: {answer.Answer}");
+        Console.WriteLine($"Confidence Score: {answer.ConfidenceScore}");
+        Console.WriteLine($"Source: {answer.Source}");
+        Console.WriteLine();
+    }
+}
+```
+
+### Conversation
+
+```C# Snippet:ConversationAnalysis_AnalyzeConversationOrchestrationPredictionConversation
+string respondingProjectName = orchestratorPrediction.TopIntent;
+TargetIntentResult targetIntentResult = orchestratorPrediction.Intents[respondingProjectName];
+
+if (targetIntentResult.TargetKind == TargetKind.Conversation)
+{
+    ConversationTargetIntentResult cluTargetIntentResult = targetIntentResult as ConversationTargetIntentResult;
+
+    ConversationResult conversationResult = cluTargetIntentResult.Result;
+    ConversationPrediction conversationPrediction = conversationResult.Prediction;
+
+    if (!String.IsNullOrEmpty(conversationResult.DetectedLanguage))
+        Console.WriteLine($"Detected Language: {conversationResult.DetectedLanguage}");
+
+    Console.WriteLine($"Top Intent: {conversationResult.Prediction.TopIntent}");
+    Console.WriteLine($"Intents:");
+    foreach (ConversationIntent intent in conversationPrediction.Intents)
+    {
+        Console.WriteLine($"Intent Category: {intent.Category}");
+        Console.WriteLine($"Confidence Score: {intent.ConfidenceScore}");
+        Console.WriteLine();
+    }
+
+    Console.WriteLine($"Entities:");
+    foreach (ConversationEntity entitiy in conversationPrediction.Entities)
+    {
+        Console.WriteLine($"Entity Text: {entitiy.Text}");
+        Console.WriteLine($"Entity Category: {entitiy.Category}");
+        Console.WriteLine($"Confidence Score: {entitiy.ConfidenceScore}");
+        Console.WriteLine($"Starting Position: {entitiy.Offset}");
+        Console.WriteLine($"Length: {entitiy.Length}");
+        Console.WriteLine();
+    }
+}
+```
+
+### LUIS
+
+```C# Snippet:ConversationAnalysis_AnalyzeConversationOrchestrationPredictionLuis
+string respondingProjectName = orchestratorPrediction.TopIntent;
+TargetIntentResult targetIntentResult = orchestratorPrediction.Intents[respondingProjectName];
+
+if (targetIntentResult.TargetKind == TargetKind.Luis)
+{
+    LuisTargetIntentResult luisTargetIntentResult = targetIntentResult as LuisTargetIntentResult;
+    BinaryData luisResponse = luisTargetIntentResult.Result;
```

Comment:
I'd suggest having a custom model that you can deserialize to, in order to prove the use case (and to give customers a sample).